Fix OpenCV libraries variable name (OpenCV-2.3 does not include backward
compat with 2.2 while OpenCV-2.4 does...)

--- cv_bridge/src/CMakeLists.txt~	2016-03-10 11:15:40.000000000 +0100
+++ cv_bridge/src/CMakeLists.txt	2016-06-14 15:56:28.401440335 +0200
@@ -2,7 +2,7 @@
 include_directories(./)
 add_library(${PROJECT_NAME} cv_bridge.cpp rgb_colors.cpp)
 add_dependencies(${PROJECT_NAME} ${catkin_EXPORTED_TARGETS})
-target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES} ${catkin_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBS} ${catkin_LIBRARIES})
 
 install(TARGETS ${PROJECT_NAME} DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
 
